pytest_girder/pytest_girder/fixtures.py

```diff
@@ -77,70 +74,18 @@ def db(request):
         pymongo.MongoClient = realMongoClient
 
 
-def _findFreePort():
-    with contextlib.closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
-        sock.bind(('', 0))
-        return sock.getsockname()[1]
+def _getPluginsFromMarker(request, registry):
+    plugins = []
```

Comment:
What is this for?

Reply:
It's a helper function for extracting the enabled plugins from pytest markers on a node

Reply:
Oh, I see.  The diff is split.  I just saw this line and was confused.